src/test/java/messenger/MessengerRequestProcessorTest.java

```diff
@@ -0,0 +1,33 @@
+package messenger;
+
+import org.junit.Test;
+
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.ObjectOutputStream;
+import java.util.HashMap;
+
+import static org.junit.Assert.*;
+
+
+public class MessengerRequestProcessorTest {
```

Comment:
What functionality does it test? Tests should closely examine their corresponding classes, this not only does not test anything, but it creates something our app uses. This shouldn't be test. This may be some groovy/bash/python script.

Reply:
It is creating java serializable file, so i think it should be in java. My reasoning was that it is only kind of fake database to show how our library is working so it should not be in source files.

Reply:
It shouldn't be in tests either. I know these directory seems like a place best suited for sth like this, but it has its sole purpose of having only tests which can be run every time you need, and testing your application. 

I would make this a groovy script and threw this at the root directory of our project, since it's only vaguely connected with app itself.